Revert erroneous calls to image_url to correct calls pix_icon

neotransotaku opened the pull request:
In reviewing the code, there are still missing reversion calls. They are covered here.

There is one slight additional change to the code in viewgiportfolio (around line 224 and 250) where I added boxing to each individual contribution to better facilitate web scraping. I am open to suggestions to delineate contributions more robustly.

davosmith commented:
Are you able to rebase your branch on top of the latest master branch? There seem to be commits in here that include fixes I've already made on the master branch.

Please also re-check your changes - some of them look like they're re-introducing calls to $OUTPUT->pix_url(), which had been correctly replaced with $OUTPUT->image_url() - it is only the calls to $OUTPUT->pix_icon() and new $pix_icon() that needed reverting back to how they were.

neotransotaku commented:
I rebased back to the last commit you made, and corrected only the erroneous `image_url` calls back to `pix_icon`. 

In the process of reviewing, I noticed that `viewcontribution.php` wraps each contribution in an `article` tag with `giportfolio-contribution` as the class. This justifies more for the changes I made in `viewgiportfolio.php` in wrapping each contribution in a `div`.

davosmith commented:
Thanks - code merged
